data/Link2.php

```diff
@@ -253,11 +253,19 @@ public function getFocus()
         return $this->focus;
     }
 
+       /**
+     * @return Array of related fields
+     */
+    function getRelatedFields(){
```

Comment:
Opening braces for methods must go on the next line according to [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)

Also you need to explicitly state the visibility of the function, in this case 'public'